packages/core/src/modules/vc/W3cVcModule.ts

```diff
@@ -8,16 +9,26 @@ import {
 
 import { SignatureSuiteRegistry, SignatureSuiteToken } from './SignatureSuiteRegistry'
 import { W3cCredentialService } from './W3cCredentialService'
+import { W3cVcModuleConfig } from './W3cVcModuleConfig'
 import { W3cCredentialRepository } from './repository/W3cCredentialRepository'
 import { Ed25519Signature2018 } from './signature-suites'
 
 export class W3cVcModule implements Module {
+  public readonly config: W3cVcModuleConfig
```

Comment:
I mostly do it to avoid having a properth with underscore, as we would then need to do:

```
class A {
  private _config

  public get config() {
    return this._cofig
  }
}
```

Readonly prevents that and is less code to do the same. Using the getter is a bit more safe I guess as readonly is not enforced at runtime